digiroad2-api-common/src/main/scala/fi/liikennevirasto/digiroad2/authentication/SessionApi.scala

```diff
@@ -30,6 +30,7 @@ class SessionApi extends ScalatraServlet {
       case Some(u) => {
         logger.info("User exists: " + username + "(" + u + ")")
         BadRequest("User exists")
+        // TODO Should we allow updating of the existing user and call userProvider.saveUser(u)?
```

Comment:
I'm thinking if it should be possible to create a user on the sessionAPI, probably it should be on the UserConfigurationApi.

Reply:
Yes, it is a bit strange that there was this kind of functionality in SessionApi.